app/models/peer_review.rb

```diff
@@ -10,8 +10,11 @@ class PeerReview < ApplicationRecord
   validates :comment, presence: true
 
   def draw_exercise(assignment)
+    regular_users = User.where(administrator: false)
+
     assignment.exercises
               .where(status: :finished)
+              .where(:user_id.in?(regular_users)) # do not give random test exercises made by creators for students to review
```

Comment:
Does this work?

Reply:
At least when I tried it.

Reply:
How about something like: `.where(user: regular_users)`

Reply:
okay so well okay erm it probably doesn't work after all :D

Reply:
that seems to work, fixing it, brb